packages/vest/src/core/test/__tests__/each.test.js

```diff
@@ -0,0 +1,132 @@
+import faker from 'faker';
+import enforce from 'n4s';
+
+import test from 'test';
+import vest from 'vest';
+
+let testObjects;
+
+describe("Test Vest's `test.each` function", () => {
+  describe('Sync Tests', () => {
+    describe('Basic Functionality', () => {
+      it('Should return all test objects correctly', () => {
+        vest.create(faker.random.word(), () => {
+          testObjects = test.each([[1, 2, 3], [2, 7, 9]])(faker.random.word(), faker.lorem.sentence(), (a, b, c) => {
+            enforce(a + b).equals(c);
+          });
+        })();
+        expect(testObjects).toHaveLength(2);
+        expect(testObjects[0].failed).toBe(false);
+        expect(testObjects[1].failed).toBe(false);
```

Comment:
Since we're here, we can also make sure that it is an actual test object.
```suggestion
        expect(testObjects[1].failed).toBe(false);
        expect(testObjects[0]).toBeInstanceOf(VestTest);
        expect(testObjects[1]).toBeInstanceOf(VestTest);
        
```